### PR TITLE
[rdf] Add linked data embed

### LINKED ISSUE #125: Embed based on linked data
Provide oembed URL for all Ginger pages, so target page’s URL can be copied and pasted into a website to show a rich snippet linking to the target website, similar to a Facebook share.

### PR DESCRIPTION
**Pull request by ddeboer**
Fix #125.

### CODE REVIEW

**Comment by ddeboer** [comment]
@finnkruyning Please add your kick-ass CSS to this PR. :wink:

Consult the changes under ‘Files changed’ for an explanation as to how the embed works.

File: modules/mod_ginger_rdf/README.md [hunk 1]
```diff
@@ -170,3 +177,25 @@ You can also use certain shortcuts to namespaced RDF properties. For instance:
...
+------
+
+This module ships with a simple embedding widget. Some JavaScript (`embed.js`)
+iterates over all `<ginger-embed>` tags on the HTML page and replaces them
+with a widget. The data is retrieved from the RDF endpoint (`controller_rdf`).
+
+To show the embed code on a resource page:
+
+```dtl
+{% include "rdf/embed.tpl %}
```
Comment (finnkruyning) [inline modules/mod_ginger_rdf/README.md]: Missing a `"`

File: modules/mod_ginger_rdf/README.md [hunk 2]
```diff
@@ -170,3 +177,25 @@ You can also use certain shortcuts to namespaced RDF properties. For instance:
...
+```
+
+Or supply a different resource id:
+
+```dtl
+{% include "rdf/embed.tpl id=123 %}
```
Comment (finnkruyning) [inline modules/mod_ginger_rdf/README.md]: Idem

File: modules/mod_ginger_rdf/lib/js/embed.js [hunk 3]
```diff
@@ -13,33 +14,56 @@
...
-                '    </picture>' +
-                '</section>';
-        }
+                html +=
+                    '    <picture>' +
+                    '        <img src="' + json['http://xmlns.com/foaf/0.1/thumbnail']['@id'] + '">' +
+                    '    </picture>';
+            }
+
+            html +='<link href="http://hart_amsterdammuseum.docker.dev/lib/css/build/ginger_embed.css" type="text/css" media="all" rel="stylesheet">' +
```
Comment (ddeboer) [inline modules/mod_ginger_rdf/lib/js/embed.js]: Must no be a hardlink

Comment (finnkruyning) [inline modules/mod_ginger_rdf/lib/js/embed.js]: So site url must be provided.

Comment (ddeboer) [inline modules/mod_ginger_rdf/lib/js/embed.js]: Or better yet: add `<link>` to the embed code?

File: modules/mod_ginger_rdf/lib/js/embed.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
+(function() {
+    'use strict';
+
+    function fetchJsonLd(url, callback, element) {
+        var xmlHttp = new XMLHttpRequest();
+        xmlHttp.onload = function(e) {
+            console.log(e.target.response);
```
Comment (ddeboer) [inline modules/mod_ginger_rdf/lib/js/embed.js]: Oops

File: modules/mod_ginger_rdf/lib/js/embed.js (hunk 3 above)
Comment (finnkruyning) [inline modules/mod_ginger_rdf/lib/js/embed.js]: Needed are three links: target page (and also widget content), site link and css link. 99% of the cases the base url will be the same.

**Pull request merged**